src/components/report-problem.component.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import React, {
+  Component,
+  View,
+  StyleSheet,
+  TouchableHighlight,
+  Text
+} from 'react-native';
+import s from './report-problem.style';
+const styles = StyleSheet.create(s);
+
+var Camera = require('react-native-camera');
+
+var RNFS = require('react-native-fs');
```

Comment:
Over the wire, but it could also be that I'm printing the data to the xcode console.  In java the output to stdout blocks and so perhaps that is why it's so slow.  I might have left a console.out in there somewhere.  But xcode output window is filled with the base64 encoded image data:  e.g.

AK6mHzI0mNyeh6iqTIkQ4kMnyY4GCKfnyg5LcggY6nn1qPLOcLwDxTlUZbdg7ep78UxJEalihTg4z0/lViMKA0YztyDzTWZjHkDDMaRZdrsW6ADjqTUdQUR1yyoCsgw7/dx1+tOcAh9mCeAMdaYr/KpJBGcEHsKkdpI8RgZK9u5z0qnqXYjEYMbRMSCF6dCPxpi/MedxwACc8GrBUswDkb8ZJ9faqkuIgrPwxbGB0NCQadSQBn+4OU9enNLtuP7qUIrBAmWZOp+n1p222/ut+dMND//W/IGFY2I8lFeRcL0KrtY4yPZe1LtTdNEuGVztxn5SU+8MdvWp/wBxIse6NvNwAju2MDso9Peop90FvNJJKgkjfazP1B9iByccVgjtULkcx2RvK6KP4QABkjpg+496rG2lS4W2DvCXzhwNpK+gA68017y3jtQZoxKJjuQAHJY+p74HNKsTtNEygPIpIVS3DRnqPY/